ingestion/src/metadata/workflow/workflow_init_error_handler.py

```diff
@@ -0,0 +1,183 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Module handles the init error messages from different workflows
+"""
+import traceback
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict, Optional, Type, Union
+
+from metadata.config.common import ConfigurationError
+from metadata.ingestion.api.parser import (
+    InvalidWorkflowException,
+    ParsingConfigurationError,
+)
+from metadata.utils.constants import UTF_8
+from metadata.utils.logger import ANSI, log_ansi_encoded_string
+
+
+class WorkflowType(Enum):
+    """
+    Workflow type enums based on the `metadata` CLI commands
+    """
+
+    INGEST = "ingest"
+    PROFILE = "profile"
+    TEST = "test"
+    LINEAGE = "lineage"
+    USAGE = "usage"
+    INSIGHT = "insight"
+    APP = "application"
```

Comment:
Should we unify this with `org.openmetadata.schema.entity.services.ingestionPipelines.PipelineType`?